contents/handbook/product/in-app-prompts.md

```diff
@@ -11,18 +11,18 @@ If you want to announce a new feature, invite a customer to a slack channel or d
 In-app prompts are powered by JSON feature flags.
 
 1. Create a new feature flag with the name starting with `prompt-`
-2. Add the payload
+2. Add the payload (delete the comments such as `// optional`)
 
    ```
```

Comment:
```suggestion
   ```typescript
```

Reply:
I've added `json` to make it clearer (previously it was typescript but it's now json)